assets/lokomotive-kubernetes/packet/flatcar-linux/kubernetes/bootstrap-configs.tf

```diff
@@ -0,0 +1,40 @@
+locals {
+  controller_bootstrap_tokens = [
+    for index in range(var.controller_count) : {
+      token_id     = random_string.bootstrap-token-id[index].result
+      token_secret = random_string.bootstrap-token-secret[index].result
+    }
+  ]
+}
+
+
+data "template_file" "bootstrap-kubeconfig" {
+  count = var.controller_count
+
+  template = file("${path.module}/workers/cl/bootstrap-kubeconfig.yaml.tmpl")
+
+  vars = {
+    token_id     = random_string.bootstrap-token-id[count.index].result
+    token_secret = random_string.bootstrap-token-secret[count.index].result
+    ca_cert      = module.bootkube.ca_cert
+    server       = "https://${local.api_server}:6443"
+  }
+}
+
+# Generate a cryptographically random token id (public)
+resource random_string "bootstrap-token-id" {
```

Comment:
Maybe we should have a module generating the tokens, so we don't duplicate so much Terraform code?